tests/consensus_tests/test_failed_snapshot_recovery.py

```diff
@@ -269,7 +271,8 @@ def test_dirty_shard_handling_with_active_replicas(tmp_path: pathlib.Path, trans
     # Delete some of the files from the shard:
     shard_path = Path(peer_dirs[-1]) / "storage" / "collections" / COLLECTION_NAME / f"{shard_id}"
     assert shard_path.exists()
-    # corrupt_shard_dir(shard_path)
+
+    corrupt_shard_dir(shard_path)
```

Comment:
Inside this function: we cannot detect a removed segment in anyway. So let's not remove a segment, but do s till remove some WAL directory.